packages/cli/src/hosting/index.js

```diff
@@ -0,0 +1,156 @@
+  }
+}
+
+async function handleError(func) {
+  try {
+    await func()
+  } catch (err) {
+    if (err && err.err) {
+      logErrorToFile(ERROR_FILE, err.err)
+    }
+    throw `Failed to start - logs written to file: ${ERROR_FILE}`
+  }
+}
+
+async function init() {
+  await checkDockerConfigured()
+  const shouldContinue = await confirmation(
+    "This will create multiple files in current directory, should continue?"
+  )
+  if (!shouldContinue) {
+    console.log("Stopping.")
+    return
+  }
+  await downloadFiles()
+  await envFile.make()
+}
+
+async function start() {
+  await checkDockerConfigured()
+  checkInitComplete()
+  console.log(info("Starting services, this may take a moment."))
+  const port = envFile.get("MAIN_PORT")
+  await handleError(async () => {
+    await compose.upAll({ cwd: "./", log: false })
+  })
+  console.log(
+    success(
+      `Services started, please go to http://localhost:${port} for next steps.`
+    )
+  )
+}
+
+async function status() {
+  await checkDockerConfigured()
+  checkInitComplete()
+  console.log(info("Budibase status"))
+  await handleError(async () => {
+    const response = await compose.ps()
+    console.log(response.out)
+  })
+}
+
+async function stop() {
+  await checkDockerConfigured()
+  checkInitComplete()
+  console.log(info("Stopping services, this may take a moment."))
+  await handleError(async () => {
+    await compose.stop()
+  })
+  console.log(success("Services have been stopped successfully."))
+}
+
+async function update() {
+  await checkDockerConfigured()
+  checkInitComplete()
+  if (
+    await confirmation(
+      "Do you wish to update you docker-compose.yaml and envoy.yaml?"
+    )
+  ) {
+    await downloadFiles()
+  }
+  await handleError(async () => {
+    const status = await compose.ps()
+    const parts = status.out.split("\n")
+    const isUp = parts[2] && parts[2].indexOf("Up") !== -1
+    if (isUp) {
+      console.log(info("Stopping services, this may take a moment."))
+      await compose.stop()
+    }
+    console.log(info("Beginning update, this may take a few minutes."))
+    await compose.pullMany(BUDIBASE_SERVICES, { log: true })
+    if (isUp) {
+      console.log(success("Update complete, restarting services..."))
+      await start()
+    }
+  })
+}
+
+const command = new Command(`${CommandWords.HOSTING}`)
+  .addHelp("Controls self hosting on the Budibase platform.")
+  .addSubOption(
+    "--init",
+    "Configure a self hosted platform in current directory.",
+    init
+  )
+  .addSubOption(
+    "--start",
+    "Start the configured platform in current directory.",
+    start
+  )
+  .addSubOption(
+    "--status",
+    "Check the status of currently running services.",
+    status
+  )
+  .addSubOption(
+    "--stop",
+    "Stop the configured platform in the current directory.",
+    stop
+  )
+  .addSubOption(
+    "--update",
+    "Updates the Budibase images to the latest version.",
```

Comment:
Major issue. This should be `Update` rather than `Updates` to keep the tense consistent. Unbelievable that you overlooked this.